src/pynwb/tests/unit/test_ernwbfile2.py

```diff
@@ -76,6 +76,15 @@ def test_add_ref(self):
         self.assertEqual(len(nwbfile.external_resources.entities), 2)
 
         self.assertEqual(nwbfile.external_resources.objects[0],
-                         (container.object_id, 'unit'))
+                         (container.object_id, 'TimeSeries/data/unit', ''))
         self.assertEqual(nwbfile.external_resources.objects[1],
-                         (table.object_id, 'test_col'))
+                         (table.columns[0].object_id, '', ''))
+
+    def test_type_map(self):
+        nwbfile = ERNWBFile(
+            session_description='session_description',
+            identifier='identifier',
+            session_start_time=datetime.datetime.now(datetime.timezone.utc)
+        )
+        _type_map = nwbfile.external_resources.type_map
+        self.assertEqual(_type_map.namespace_catalog.namespaces, ('hdmf-common', 'hdmf-experimental', 'core', 'ndx-external-resources'))
```

Comment:
This would be a more direct test if it works. Can you check it?

```suggestion
        from pynwb import get_type_map
        self.assertEqual(_type_map, get_type_map())
```

Reply:
I ran it and it doesn't work. Two different type maps.

Reply:
Never mind then. This is good. Thanks!